macro-pdfviewer-api/src/main/resources/templates/pdfviewer/pdfviewer.vm

```diff
@@ -124,24 +38,25 @@
       <div class="box warningmessage">PDF Viewer : Not compatible with IE < 10</div>
     <![endif]-->
     <![if !(lt IE 10)]>
-      <iframe class="pdfviewer" src="${pdfviewerurl}?${queryString}#locale=${xcontext.locale}" width="${width}"
-        height="${height}">
+      <iframe class="pdfviewer" src="${pdfviewerurl}?${queryString}#locale=${xcontext.locale}" width="
+        $escapetool.xml($width)" height="$escapetool.xml($height)" data-delegated="$file.areViewRightsDelegated()">
```

Comment:
I prefer to always escape the attribute values, to be safe, even if the expected return type (but you never know how this could be overwritten) appears to be safe.